go/support_infrastructure/domain_service/domain_service.go

```diff
@@ -226,17 +229,17 @@ func parseSaysStatement(saysStatement *auth.Says) (*auth.Prin, *auth.Prin, *auth
 // Tao name of the program and subject public key being programKey.
 // Certificate expiration time is one year from issuing time.
 func GenerateProgramCert(domain *tao.Domain, serialNumber int, programPrin *auth.Prin,
-	programKey *auth.Prin, now, expiry time.Time) (*tao.Attestation, error) {
+	programKey *tao.Keys, now, expiry time.Time) (*tao.Attestation, error) {
 
 	policyCert := domain.Keys.Cert
 	x509Info := domain.Config.GetX509Info()
 	programName := programPrin.String()
 	x509Info.CommonName = &programName
 	subjectName := tao.NewX509Name(x509Info)
-	verifier, err := tao.FromPrincipal(*programKey)
-	if err != nil {
-		return nil, err
-	}
+	verifier := programKey.VerifyingKey
+	//if err != nil {
```

Comment:
I've seen a lot of commenting out of code in the latest commits. Please use source control to store code instead of stashing it in comments.